docs/content/mongocxx-v3/installation/linux.md

```diff
@@ -9,30 +9,55 @@ title = "Linux"
 
 ### Step 1: Choose a C++17 polyfill
 
-The mongocxx driver uses the C++17 features `std::optional` and
-`std::string_view`. To compile the mongocxx driver for pre-C++17, you
-must choose one of the following implementations for these features:
-
-   MNMLSTC/core (*default for non-Windows platforms*)
-     Select with `-DBSONCXX_POLY_USE_MNMLSTC=1`.  **NOTE**: This option
-     vendors a header-only installation of MNMLSTC/core into the bsoncxx
-     library installation and will therefore download MLNMLSTC from GitHub
-     during the build process. If you already have an available version of
-     MNMLSTC on your system, you can avoid the download step by using
-     `-DBSONCXX_POLY_USE_SYSTEM_MNMLSTC`.
-
-   Boost (*default for Windows platforms*)
-     Select with `-DBSONCXX_POLY_USE_BOOST=1`. This is currently the
-     only option if you are using a version of MSVC that does not support
-     C++17.
-
-Most users should be fine sticking with the default. However, if you
-have an existing application which makes heavy use of one of the
-available libraries, you may prefer to build the mongocxx driver
-against the same library.
-
-**DO NOT** change your project's polyfill if you need to create a
-stable binary interface.
+The mongocxx driver uses C++17 features `std::optional` and `std::string_view`.
```

Comment:
Could this section -- repeated between `{linux,macos,windows}.md` -- be refactored into a separate page?

Reply:
Good idea. Added a new `mongocxx-v3/polyfill-selection` page that is linked to from the per-OS installation pages.